tools/database/src/utils.rs

```diff
@@ -0,0 +1,26 @@
+use std::path::Path;
+
+pub(crate) fn open_rocksdb(home: &Path) -> anyhow::Result<near_store::db::RocksDB> {
+    let config = nearcore::config::Config::from_file_skip_validation(
+        &home.join(nearcore::config::CONFIG_FILENAME),
+    )?;
+    let store_config = &config.store;
+    let db_path = store_config.path.as_ref().cloned().unwrap_or_else(|| home.join("data"));
+    let rocksdb = near_store::db::RocksDB::open(
+        &db_path,
+        store_config,
+        near_store::Mode::ReadOnly,
+        near_store::Temperature::Hot,
+    )?;
+    Ok(rocksdb)
+}
+
+pub(crate) fn updated_displayed_progress(i: usize, n: usize) {
+    if (i + 1) * 100 % n == 0 {
+        eprint!("\rProgress: {}%", (i + 1) * 100 / n);
+        std::io::Write::flush(&mut std::io::stderr()).unwrap();
+        if i == n - 1 {
+            eprintln!();
+        }
+    }
```

Comment:
Maybe replace this with `indicatift` library, since we already use it in our codebase

Reply:
ah, great suggestion, replaced!